pkg/vdr/peer/vdr.go

```diff
@@ -36,6 +38,16 @@ func New(s storage.Provider) (*VDR, error) {
 	return &VDR{store: didDBStore}, nil
 }
 
+// Update did doc.
+func (v *VDR) Update(didDoc *diddoc.Doc, opts ...vdrapi.DIDMethodOption) error {
+	return fmt.Errorf("not supported")
+}
+
+// Deactivate did doc.
+func (v *VDR) Deactivate(did string, opts ...vdrapi.DIDMethodOption) error {
+	return fmt.Errorf("not supported")
+}
+
 // Accept did method.
```

Comment:
How does VDR indicate that it can handle only resolution vs the full interface?

Reply:
I think we need to consider having two sets of VDRs loaded into the framework:
- resolvers
- writers / full lifecycle

and make the writer interface a superset of the resolver interface.

Reply:
@troyronda I added TODO https://github.com/hyperledger/aries-framework-go/compare/d0b6dc3cd24afcba1367852c0fa823403a7fa66d..8ec7bd065b96b9cd25ffe5ee81be5a954167d77b#diff-48c6230240bd8b22d01c4586d81f82976e4c505ba3b01c270f82400255f83412R34